examples/experimental/horizontal-pod-autoscaling/README.md

```diff
@@ -0,0 +1,207 @@
+php-apache   run=php-apache   run=php-apache             80/TCP
+```
+
+Now, we will wait some time and verify that both the replication controller and the service were correctly created and are running. We will also determine the IP address of the service:
+
+```console
+$ kubectl get pods
+NAME               READY     STATUS    RESTARTS   AGE
+php-apache-wa3t1   1/1       Running   0          12m
+
+$ kubectl describe services php-apache | grep "LoadBalancer Ingress"
+LoadBalancer Ingress:	146.148.24.244
+```
+
+We may now check that php-apache server works correctly by calling ``curl`` with the service's IP:
+
+```console
+$ curl http://146.148.24.244
+OK!
+```
+
+Please notice that when exposing the service we assumed that our cluster runs on a provider which supports load balancers (e.g.: on GCE).
+If load balancers are not supported (e.g.: on Vagrant), we can expose php-apache service as ``ClusterIP`` and connect to it using the proxy on the master:
+
+```console
+$ kubectl expose rc php-apache --port=80 --type=ClusterIP
+NAME         LABELS           SELECTOR         IP(S)     PORT(S)
+php-apache   run=php-apache   run=php-apache             80/TCP
+
+$ kubectl cluster-info | grep master
+Kubernetes master is running at https://146.148.6.215
+
+$ curl -k -u admin:password https://146.148.6.215/api/v1/proxy/namespaces/default/services/php-apache/
+OK!
+```
+
+
+## Step Two: Create horizontal pod autoscaler
+
+Now that the server is running, we will create a horizontal pod autoscaler for it.
+To create it, we will use [hpa-php-apache.yaml](hpa-php-apache.yaml) file, which look like this:
+
+```yaml
+apiVersion: experimental/v1alpha1
+kind: HorizontalPodAutoscaler
+metadata:
+  name: php-apache
+  namespace: default
+spec:
+  maxReplicas: 10
+  minReplicas: 1
+  scaleRef:
+    kind: ReplicationController
+    name: php-apache
+    namespace: default
+  target:
+    quantity: 100m
+    resource: cpu
+```
+
+Please be aware that it sets the target CPU consumption to 100 milli-cores, while in [rc-php-apache.yaml](rc-php-apache.yaml) each pod requests 200 milli-cores.
+As a general rule, the autoscaler's target should be lower than the request.
+Otherwise, overloaded pods may not be able to consume more resources than requested what will prevent scaling-up.
+
+We will create the autoscaler by executing the following command:
+
+```console
+$ kubectl create -f ./examples/experimental/horizontal-pod-autoscaling/hpa-php-apache.yaml
+horizontalpodautoscaler "php-apache" created
+```
+
+We may check the current status of autoscaler by running:
+
+```console
+$ kubectl get experimental/hpa
+NAME         REFERENCE                                   TARGET     CURRENT   MINPODS   MAXPODS   AGE
+php-apache   ReplicationController/default/php-apache/   100m cpu   0 cpu     1         10        4m
+```
+
+Please not that the current CPU consumption is 0 as we are not sending any requests to the server.
```

Comment:
s/not/note/
Also, explain how the "current" column is computed (IIRC it's just the average across all the pods controlled by the corresponding RC?)